### DISCUSSION TITLE
Hackathon 18-05-22

### DISCUSSION THREAD

**Discussion by w4xccino** [w4xccino/peepo-app#46] [Show and tell]
Primera hackathon del proyecto, quiza la unica. JUnto a @romariou76

**Comment by w4xccino** [comment]
## My goals

- [x] Agregar boton `Home` en la NavBar
- [x] Api productos
- [x] Rutas dinamicas

**Comment by romariou76** [comment]
## My goals
- [ ] Listar datos de usuario al momento de logearse
- [ ] Crear la pagina principal vistosa
